2022.md

```diff
@@ -2,6 +2,8 @@
 
 ## June (1)
```

Comment:
Update the count here

Reply:
Resolved [9e0a1e7](https://github.com/wso2/iam-blogs/pull/130/commits/9e0a1e751f10f8f72d346cced000be7011204f38)